src/components/layout.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactChildren, FC } from "react"
+import { useStaticQuery, graphql } from "gatsby"
+import { ThemeProvider } from "styled-components"
+
+import Header from "./header/header"
+import theme from "../theme"
+import GlobalStyle from "../styles/Global"
+
+interface LayoutProps {
+  children: ReactChildren
+}
+
+const Layout: FC<LayoutProps> = ({ children }) => {
+  const data = useStaticQuery(graphql`
+    query SiteTitleQuery {
+      site {
+        siteMetadata {
+          title
+        }
+      }
+    }
+  `)
+
+  return (
+    <ThemeProvider theme={theme}>
+      <>
```

Comment:
ThemeProvider only accepts a single child element, so we need to add the fragments here btw!

Reply:
Oh I see!!!